packages/11ty/_plugins/transforms/outputs/pdf/transform.js

```diff
@@ -82,7 +82,8 @@ module.exports = function(eleventyConfig, collections, content) {
         setDataAttributes(currentPage, sectionElement)
 
         // set an id for anchor links to each section
-        sectionElement.setAttribute('id', mainElement.dataset.pageId)
+        const computedPageID = `page_${mainElement.dataset.pageId}`
+        sectionElement.setAttribute('id', computedPageID)
```

Comment:
The value of `mainElement.dataset.pageId` already has the page-prefixed since it is pulled from the template generated by `base.11ty.js`, so you don't need to re-compute it here
```suggestion
        sectionElement.setAttribute('id', mainElement.dataset.pageId)
```

Reply:
Thanks @hbalenda , I've made these changes.